### PR TITLE
Final PR for the models page

### PR DESCRIPTION
**Pull request by Yukino2002**
Has a few minor path fixes, workflow edit, and refactoring the stylesheet structure.

### CODE REVIEW

File: new-website/deepchem/styles/models.css [hunk 1]
```diff
@@ -1,48 +0,0 @@
-@tailwind base;
```
Comment (rbharath) [inline new-website/deepchem/styles/models.css:1]: Why are we deleting this CSS file?

Comment (rbharath) [reply new-website/deepchem/styles/models.css:1]: It looks like we moved this over to globals.css is that right?

**Review by rbharath — COMMENTED** [review]
A couple of basic questions on the PR

File: new-website/deepchem/styles/models.css (hunk 1 above)
Comment (Yukino2002) [reply new-website/deepchem/styles/models.css:1]: Yes, we have shifted the models.css content to global.css. I am not sure exactly why, but the previous structure caused some of the styling of the homepage to be overwritten. Shifting the same solves the problem.

Comment (Yukino2002) [reply new-website/deepchem/styles/models.css:1]: I have been trying to find the exact reason for this issue, but found nothing concrete till now.

Comment (rbharath) [reply new-website/deepchem/styles/models.css:1]: Ok sounds good. Can you add a screenshot for the rendered site to this issue? I think we should be good to merge after

Comment (Yukino2002) [reply new-website/deepchem/styles/models.css:1]: In the previous version, the explore heading is not being styled correctly

Explore-heading-not-properly-rendered

The expected styling is

correct-rendering

**Review by rbharath — APPROVED** [review]
LGTM

**Pull request merged**